File: PHP/UberVan.php
<?php

require_once('Car.php');

class UberVan extends Car
{
    public $typeCarAccepted;
    public $seatsMaterial;

    public function UberVan($licence, $driver, $typeCarAccepted, $seatsMaterial, $passenger)
    {
        parent::__construct($licence, $driver, $passenger);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }
}

File: Python/uberVan.py
from car import Car

class UberVan(Car):
   
   typeCarAccepted = []
   seatsMaterial   = []
   
   def __init__(self,license, driver, typeCarAccepted, seatsMaterial)
       super. __init__(self,license,driver)
       self.typeCarAccepted = typeCarAccepted
       self.seatsMaterial   = seatsMaterial